index.html

```diff
@@ -39,23 +37,128 @@ <h2>New Entry</h2>
             <label for="photo">Photo URL</label>
             <input required="required" name="photoUrl" type="text" id="photo" class="photo">
           </div>
-        </div> <!--row div-->
+        </div>
+
         <div class="row col-1">
           <label for="notes">Notes</label>
           <textarea required="required" class="note" name="notes" rows="10" id="notes"></textarea>
+          <div class="space-around">
+
+            <div class="delete visitibility-hidden"><a href="#">Delete Entry</a>
+            </div>
+
+            <div class="modal hidden">
+              <div class="modal-content hidden row justify-center">
+                Are you sure you want to delete this entry?
+                <div class="space-around margin-text-entry">
+                  <button class="gray cancel">CANCEL</button>
+                  <button class="orange confirm">CONFIRM</button>
+                </div>
+              </div>
+            </div>
+
+            <button class="button-save">SAVE</button>
+          </div>
 
-          <button class="button-save">SAVE</button>
         </div>
 
       </form>
-    </div> <!--entry form div-->
+    </div>
 
     <div data-view="entries" class="second">
-      <div class="row margin-left entries">
+
+      <div class="margin-left entries space-around">
         <h2>Entries</h2>
-        <button class="margin-button new">NEW</button>
+        <button class="new">NEW</button>
+      </div>
+
+      <p class="text-center">No entries have been recorded</p>
+
+      <ul class="list">
+      </ul>
+
+    </div>
+  </main>
+  <script src="js/data.js"></script>
+  <script src="js/main.js"></script>
+</body>
+</html> -->
```

Comment:
I highly suggest removing your duplicate commented-out html content if you have no plans on using it again.

Reply:
i'll fix it. thank you.